boards/ssrc/saluki/minimal.px4board

```diff
@@ -0,0 +1,80 @@
+CONFIG_BOARD_TOOLCHAIN="riscv64-unknown-elf"
+CONFIG_BOARD_ARCHITECTURE="rv64gc"
+CONFIG_BOARD_SERIAL_GPS1="/dev/ttyS4"
+CONFIG_BOARD_SERIAL_TEL1="/dev/ttyS1"
+CONFIG_BOARD_SERIAL_RC="/dev/ttyS2"
+CONFIG_BOARD_ROMFSROOT="minimal_romfs"
+CONFIG_COMMON_BAROMETERS=n
+CONFIG_DRIVERS_BATT_SMBUS=n
+CONFIG_COMMON_DISTANCE_SENSOR=n
+CONFIG_DRIVERS_GPS=n
+CONFIG_DRIVERS_IMU_INVENSENSE_ICM42688P=n
+CONFIG_DRIVERS_IMU_INVENSENSE_ICM20649=n
+CONFIG_DRIVERS_MAGNETOMETER_BOSCH_BMM150=y
```

Comment:
needed?

Reply:
px4 work queue now only links against px4_layer via some module. So I just added some module using the work queue into the build, to work around this problem.
It is not possible to link the work queue directly agains px4_layer, because in protected build (px4_fmuv5_protected target) there are two versions of px4_layer, one for kernel and one for user side. but the work queue is a common library for both....